localization/pose_estimator/ndt_scan_matcher/ndt_omp/apps/align.cpp

```diff
@@ -45,17 +44,16 @@ pcl::PointCloud<pcl::PointXYZ>::Ptr align(
   registration->setInputTarget(target_cloud);
   registration->setInputSource(source_cloud);
   pcl::PointCloud<pcl::PointXYZ>::Ptr aligned(new pcl::PointCloud<pcl::PointXYZ>());
-
-  auto t1 = ros::WallTime::now();
+  auto t1 = std::chrono::steady_clock::now();
   registration->align(*aligned);
-  auto t2 = ros::WallTime::now();
-  std::cout << "single : " << (t2 - t1).toSec() * 1000 << "[msec]" << std::endl;
+  auto t2 = std::chrono::steady_clock::now();
+  std::cout << "single : " << std::chrono::duration_cast<std::chrono::milliseconds>(t2 - t1).count() << "[msec]" << std::endl;
 
   for (int i = 0; i < 10; i++) {
     registration->align(*aligned);
   }
-  auto t3 = ros::WallTime::now();
-  std::cout << "10times: " << (t3 - t2).toSec() * 1000 << "[msec]" << std::endl;
+  auto t3 = std::chrono::steady_clock::now();
+  std::cout << "10times: " << std::chrono::duration_cast<std::chrono::milliseconds>(t3 - t2).count() << "[msec]" << std::endl;
```

Comment:
We should use the logging facilities instead of `cout`

Reply:
@nnmm Also if output is necessary perhaps should combine output with previous cout (single and 10x's time in one output/log)?

Reply:
@fred-apex-ai This code is independent from rclcpp, why would `std::cout` not be suitable here?